core/src/player.rs

```diff
@@ -526,10 +532,12 @@ impl Player {
             });
 
             self.update_timers(dt);
+            self.update(|context| {
```

Comment:
Hey, I know this is super duper old, but: any reason this uses `update`, and not `mutate_with_update_context`?
Asking because the former does way more things (like triggering GC, which in practice does nothing due to incremental logic), and on desktop I saw it being called thousands of times per frame. If this _is_ needed for some specific part (queued avm1 actions?), I could try refactoring this to only do that part. If it's not needed, I'll make a PR to replace it with `mutate_with_update_context`.